stable/insights-agent/values.yaml

```diff
@@ -609,6 +609,13 @@ right-sizer:
 right-sizer-vpa:
   admissionController:
     registerWebhook: true
+    image:
+      # custom Fairwinds vpa-admission-controller image
+      repository: quay.io/fairwinds/vpa-admission-controller
+      tag: master
```

Comment:
I assume `master` tag is dynamic, is that what we want?

Reply:
yes, we probably may want improve this in future